<launch>
    <node pkg="simple_task" type="simple_task_node" name="simple_task_node" output="screen">
        <param name="odom_topic" value="/mobile_base_controller/odom" />
        <param name="cmd_vel_topic" value="/cmd_vel" />
        <param name="linear_speed" value="0.2" />       <!-- m/s -->
        <param name="angular_speed" value="0.3" />      <!-- ras/s -->
        <param name="linear_error" value="0.015" />     <!-- m -->
        <param name="angular_error" value="2" />        <!-- degree -->
    </node>
</launch>